ihaskell-display/ihaskell-aeson/IHaskell/Display/Aeson.hs

```diff
@@ -16,4 +16,4 @@ instance IHaskellDisplay Value where
   display renderable = return $ Display [plain json, html dom]
     where
       json = T.unpack $ E.decodeUtf8 $ LBS.toStrict $ encodePretty renderable
-      dom = [i|<div class="highlight-code" id="javascript">${json}</div>|]
+      dom = [i|<div class="highlight-code" id="javascript">${json}</div>|]
```

Comment:
Can you add a newline to the end of these files? It creates unnecessary diff changes. (`hindent` skips that newline, but generally I prefer to have that newline. I use `hindent` from inside an editor, which avoids that, but you could also just re-add that newline after piping stuff through hindent.)